docs/plugins.md

```diff
@@ -518,7 +518,7 @@ export default {
 
 ### Utility functions
 
-A `util` module from Prettier core is considered a private API and is not meant to be consumed by plugins. Instead, the `util-shared` module provides the following limited set of utility functions for plugins:
+A `util` module from Prettier core is considered a private API and is not meant to be consumed by plugins. Instead, the `util` object from Prettier provides the following limited set of utility functions as properties for plugins:
```

Comment:
Let's remove 


> A `util` module from Prettier core is considered a private API and is not meant to be consumed by plugins.

This is confusing.

Reply:
Let me get sure that just improving docs/plugins.md is enough and I don't need to update https://github.com/prettier/prettier/blob/main/docs/api.md since its not for plugin author but for users who want to run Prettier programmatically.

Reply:
```suggestion
`prettier.util` provides the following limited set of utility functions for plugins:
```

Reply:
Thank you, I committed the suggestion.